### ISSUE TITLE
[terra-clinical-item-view] insufficient color-contrast for secondary content on hover/focus background

### ISSUE THREAD

**Issue by benbcai** [cerner/terra-clinical#673]
# Bug Report

## Description
When a secondary item view content  is placed in a terra-list there is insufficient color-contrast (only for fusion-theme) between color of a secondary item view content and the selected hover/focus background of the list. This is happening in Firefox/IE but works in Chrome.

The fusion-theme color for [--terra-clinical-item-view-content-secondary-color](https://github.com/cerner/terra-clinical/blob/821a6f560c7ce0f9ab2fb50f101d198506822313/packages/terra-clinical-item-view/src/orion-fusion-theme/ItemView.module.scss#L5) is `#64696c`. The fusion-theme for [--terra-list-item-selected-focus-background-color](https://github.com/cerner/terra-core/blob/e7ca384ee08d77ec69574ca2682f403676468cf1/packages/terra-list/src/orion-fusion-theme/List.module.scss#L28-L29) is `#cbe7fa`. The contrast ratio for these colors is `4.32 : 1`. 

## Steps to Reproduce

1. Go to the [selectable item collection test](https://engineering.cerner.com/terra-clinical/tests/terra-clinical-item-collection/clinical-item-collection/item-collection-selectable).
2. Resize to a small breakpoint to show the list view.
3. Select and hover over the first item.
4. Inspect the secondary content ("Display2") and its background.

## Additional Context / Screenshots
![Screen Shot 2020-06-12 at 3 58 18 PM](https://user-images.githubusercontent.com/22618655/84547988-43422100-acca-11ea-8513-f186122e9582.png)

```
1) [medium] Item Collection Returns child key for click when onSelect is given to a selectable item collection [default] is accessible and is within the mismatch tolerance:
expected no accessibility violations but got:
	{
  "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
  "help": "Elements must have sufficient color contrast",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=axeAPI",
  "id": "color-contrast",
  "impact": "serious",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": {
            "bgColor": "#cbe7fa",
            "contrastRatio": 4.32,
            "expectedContrastRatio": "4.5:1",
            "fgColor": "#64696c",
            "fontSize": "10.5pt (14px)",
            "fontWeight": "normal",
            "messageKey": null
          },
          "id": "color-contrast",
          "impact": "serious",
          "message": "Element has insufficient color contrast of 4.32 (foreground color: #64696c, background color: #cbe7fa, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1",
          "relatedNodes": [
            {
              "html": "<li tabindex=\"0\" id=\"item1\" role=\"option\" aria-selected=\"true\" data-item-show-focus=\"true\" class=\"List-module__list-item___1Gt4b List-module__selected___32shv List-module__is-selectable___3PyDe\">",
              "target": [
                "#item1"
              ]
            }
          ]
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 4.32 (foreground color: #64696c, background color: #cbe7fa, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1",
      "html": "<div data-terra-clinical-item-display-text=\"true\" class=\"ItemDisplay-module__text___c-QUG\">Display 2</div>",
      "impact": "serious",
      "none": [],
      "target": [
        "#item1 > .List-module__item-fill___3VrIn > .ItemView-module__item-view___61CR3.ItemView-module__one-column___2LPcY > .ItemView-module__body___SD0Vl > .ItemView-module__row-container___3YCG9 > .ItemView-module__row___1_JaL:nth-child(3) > .ItemView-module__content-secondary-color___3j2Dv.ItemView-module__content-secondary-size___2MzM9.ItemView-module__content___3wPtn > .ItemDisplay-module__item-display___3yi9q > .ItemDisplay-module__text___c-QUG[data-terra-clinical-item-display-text=\"true\"]"
      ]
    }
  ],
  "tags": [
    "cat.color",
    "wcag2aa",
    "wcag143"
  ]
}
```

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):

**Comment by benbcai** [comment]
Since this is a cross-component insufficient color-contrast violation and each component cannot enforce or predict what styles are used by its HOC or children, is this something that we can correct or is it better to just ignore this type of cross-component color-contrast violation?

**Comment by saket2403** [comment]
--terra-clinical-item-view-content-secondary-color is being updated to a new value as per this [comment](https://github.com/cerner/terra-clinical/pull/534#issuecomment-638872302) (last point) in PR #534. The new ratio between `#cbe7fa` and `rgba(28, 31, 33, 0.65)` (new value) is 4.65 which passes color contrast. cc @benbcai

**Comment by neilpfeiffer** [comment]
In auditing the Fusion color - I discovered that the original hex value `#64696c` (and therefore also the new RGBA value `rgba(28, 31, 33, 0.65)`) was not the actual color commonly used by Fusion, although it is great that the RGBA value does pass, which is why we switched from HEX to RGBA. 

The actual color Fusion uses most commonly for secondary text is/was supposed to be `#595f62`, which translates for Terra to `rgba(28, 35, 40, 0.73)`. This color is darker, so will equally pass color contrast as well. @saket2403 I updated the color in PR#534 and regenerated screenshots, so this is now resolved and will merge in with #534
